CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# How To Contribute
+
+## Some Ways to Contribute
+
+- Report potential bugs
```

Comment:
The information on how to do this is missing from this document.  For example, there shouldn't be any need to clone the repo to report a bug (e.g., a documentation issue), or to suggest product enhancements.   There should instead be guidance around how to file a github issue and what the expectations are.   See https://github.com/microsoft/ebpf-for-windows/blob/master/CONTRIBUTING.md for an example (which we got from other projects, so we take no credit for that).

Furthermore, is there a security vulnerability reporting process?